code/src/Components/YourName.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const YourName = ({ yourName, onNameChange }) => {
+
+  return (
+  <div className="name">
+    <h3 tabIndex="0">YOUR NAME</h3>
+    <label htmlFor="yourName"></label>
+    <input value={yourName} onChange={e => onNameChange(e.target.value)} id="yourName">
+    </input>
+  </div>
+  )
+}
```

Comment:
Comment from lighthouse report in dev tools: "Heading elements are not in a sequentially-descending order". There is a H1 and a H3 but no H2. I think you would avoid this message if you make the h3 in to a h2 instead.